src/recipes/heartbeat-by-user-first-impression/config.js

```diff
@@ -33,24 +35,40 @@ let supportedLocales = [ 'de',
 ];
 
 module.exports = {
-  "nightly": {
-    restdays: 30,
-    sample: 1 / thousand,  // 1 of 1000
-    locales: supportedLocales
+  VERSION: VERSION,
+
+  surveyUrls: {
```

Comment:
surveyUrl is currently assumed to be 1:1 with surveyBreaks.  Can we make these one data structure or add a TODO?
e.g.
surveys: {
  <locale>: ['url': <url>, 'break': <break>]
}